pipelines/docker-build-rhtap/patch.yaml

```diff
@@ -119,3 +119,8 @@
     taskRef:
       kind: Task
       name: acs-deploy-check
+- op: add
+  path: /spec/results/-
+  value:
+    name: SBOM_BLOB_URL
+    value: "$(tasks.build-container.results.SBOM_BLOB_URL)"
```

Comment:
> needs to be properly added to the pipeline as well.

Why?

Reply:
From my testing, if the result is not added to the pipeline, it does not show in the Openshift UI. I didn't try to retrieve it via CLI, so I just assumed it was not available at all.

Reply:
It will be available on the TaskRun yaml, which EC does have access to. The result doesn't really seem like a user-facing one (unless it can serve as the LINK_TO_SBOM as well?)

Reply:
Good point, so I'll drop this commit.